javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -0,0 +1,260 @@
+package java.util
+
+import scala.collection.mutable
+
+class LinkedList[E]() extends AbstractSequentialList[E]
+    with List[E] with Deque[E] with Cloneable with Serializable { self =>
+
+  def this(c: Collection[_ <: E]) = {
+    this()
+    addAll(c)
+  }
+
+  val inner: mutable.ListBuffer[Box[E]] = new mutable.ListBuffer()
+
+  def getFirst(): E =
+    if (inner.isEmpty) throw new NoSuchElementException()
+    else inner.head.inner
```

Comment:
For a simple one liner you could do `inner.headOption.getOrElse(throw new NoSuchElementException)`